administrator/modules/mod_login/tmpl/default.php

```diff
@@ -25,7 +25,7 @@
 							<?php echo JText::_('JGLOBAL_USERNAME'); ?>
 						</label>
 					</span>
-					<input name="username" tabindex="1" id="mod-login-username" type="text" class="input-medium" placeholder="<?php echo JText::_('JGLOBAL_USERNAME'); ?>" size="15" autofocus="true" data-tests="username"/>
```

Comment:
I'm removing this HTML5 attribute that was added with this idea: https://github.com/joomla/joomla-cms/pull/10589

But the current agreement is that we are not going to use this attributes unless is really needed. So for the moment we will use the existing IDs for locating the elements and use the AdministratorLogin Page Object to store the locators.